Scripts/Core/Contexts/EditorVRContext.cs

```diff
@@ -13,24 +13,31 @@ class EditorVRContext : ScriptableObject, IEditingContext
         [SerializeField]
         float m_RenderScale = 1f;
 
+        [SerializeField]
+        bool m_CopySceneCameraSettings = true;
+
         [SerializeField]
         internal List<MonoScript> m_DefaultToolStack;
 
         EditorVR m_Instance;
 
+        public bool copySceneCameraSettings { get { return m_CopySceneCameraSettings; } }
+
         public void Setup()
         {
             EditorVR.defaultTools = m_DefaultToolStack.Select(ms => ms.GetClass()).ToArray();
             m_Instance = ObjectUtils.CreateGameObjectWithComponent<EditorVR>();
-
             XRSettings.eyeTextureResolutionScale = m_RenderScale;
         }
 
         public void Dispose()
         {
-            m_Instance.Shutdown(); // Give a chance for dependent systems (e.g. serialization) to shut-down before destroying
-            ObjectUtils.Destroy(m_Instance.gameObject);
-            m_Instance = null;
+            if (m_Instance)
```

Comment:
Why the null check?

Reply:
I ran into a case that produced an exception when calling Dispose() before the EXR instance is created in IEditingContext's Setup() function.  I've just pushed an alternate fix that validates that an instance has been created, via the instanceExists bool prop in IEditingContext, before attempting to dispose in EditingContextManager's SetEditingContext() function.